src/common/components/transfer/index.tsx

```diff
@@ -377,26 +420,29 @@ export class Transfer extends Component<Props, State> {
         const fullAmount = `${amount} ${asset}`;
         const username = activeUser?.username!
 
-        let prms: Promise<any>;
         switch (mode) {
-            case 'transfer':
+            case "transfer":
                 if (asset === "POINT") {
                     transferPointHot(username, to, fullAmount, memo);
                 } else {
-                    prms = transferHot(username, to, fullAmount, memo);
+                    transferHot(username, to, fullAmount, memo);
                 }
                 break;
-            case 'transfer-saving':
-                prms = transferToSavingsHot(username, to, fullAmount, memo);
+            case "transfer-saving":
+                transferToSavingsHot(username, to, fullAmount, memo);
                 break;
-            case 'convert':
-                prms = convertHot(username, fullAmount)
+            case "convert":
+                convertHot(username, fullAmount)
                 break;
-            case 'withdraw-saving':
-                prms = transferFromSavingsHot(username, to, fullAmount, memo);
+            case "withdraw-saving":
+                transferFromSavingsHot(username, to, fullAmount, memo);
                 break;
-            case 'power-up':
-                prms = transferToVestingHot(username, to, fullAmount);
+            case "power-up":
+                transferToVestingHot(username, to, fullAmount);
+                break;
+            case "delegate":
+                const vests = this.hpToVests(Number(amount), false);
```

Comment:
`withSymbol` should be true otherwise we would get serialization issue.

Reply:
hive signer doesn't accept with amount with `VESTS` that's why withSymbol exists in that function.

Reply:
@talhasch Really, I tried without VESTS with hivesigner, it said serialization issue.

Reply:
here screenshot of error: 
![image](https://user-images.githubusercontent.com/3015990/101884999-2ff10f00-3ba2-11eb-8b29-d7d4ea431b2a.png)

Reply:
ok let me check it